nengo_bones/templates/base_script.sh.template

```diff
@@ -7,6 +7,22 @@ NAME=$0
 COMMAND=$1
 STATUS=0  # used to exit with non-zero status if any command fails
 
+# set some useful environment variables for interoperability between github actions
+# and travis ci
+# shellcheck disable=SC2034
```

Comment:
No strong reason, I thought it might be good practice to avoid exporting if we don't need it but I can't foresee any actual scenarios where exporting these would cause issues :+1:.